code/core/src/main/java/com/adobe/marketing/mobile/rulesengine/RulesEngine.java

```diff
@@ -20,7 +21,7 @@
     private final Object rulesEngineMutex = new Object();
     private final Evaluating evaluator;
     private final Transforming transformer;
-    private List<T> rules;
+    @VisibleForTesting() public List<T> rules;
```

Comment:
Can you expose a member function `getRules()` which returns a copy of the internal rules object instead of making the variable public? I also don't recommend `@VisibleForTesting` for public members.